pkgs/development/tools/packet/default.nix

```diff
@@ -3,12 +3,12 @@
 
 buildGoPackage rec {
   pname = "packet";
-  version = "v2.2.2";
+  version = "2.2.2";
 
   goPackagePath = "github.com/ebsarr/packet";
 
   src = fetchgit {
```

Comment:
A candidate for fetchfromGitHub.

Reply:
i just noticed, the header in the file: "# This file was generated by https://github.com/kamilchm/go2nix v1.2.1"
maybe we should better revert these changes, because they will get overwritten in the future.

Reply:
sorry I did not apply the changes yet :see_no_evil:

Reply:
go2nix is only used for first generation and is kinda outdated/deprecated. This change is fine and also changing to fetchFromGitHub. Also the comment can be removed.